### DISCUSSION TITLE
SolarEdge Home Load Controller EM-DCS-R08-00

### DISCUSSION THREAD

**Discussion by gitmawa** [WillCodeForCats/solaredge-modbus-multi#650] [Q&A] [answered]
Is the [Solaredge Load Home Controller EM-DCS-R08-00](https://knowledge-center.solaredge.com/sites/kc/files/se-home-load-controller-datasheet-row.pdf) supported in the Home Assistant? I can see the device in my Solaredge app, but the device is not displayed in "SolarEdge Modbus Multi".

![image](https://github.com/user-attachments/assets/e1498d8e-1fc1-42d2-9282-425cff940add)

![image](https://github.com/user-attachments/assets/5223bf98-b726-4081-a757-43167ec34bab)

![image](https://github.com/user-attachments/assets/c76a2cac-b380-4259-b4ba-78b7c23c81ce)

**Comment by WillCodeForCats** [comment, marked as answer]
No it's not.

For more information read https://github.com/WillCodeForCats/solaredge-modbus-multi/discussions/285
